.github/CODEOWNERS

```
@@ -1,3 +1,3 @@
 # These owners will be the default owners for everything in
 # the repo
-*   @Serdaro @rebekah-lawrence @oliverhowell
+*   @hazelcast/documentation
```

@JackPGreen can we add CN team also? 
```suggestion
*   @hazelcast/documentation @hazelcast/cloud-native
```

---

it is not must have, I just want to check it 🙂